src/sdkAnalytics/getSDKAnalyticsSignature.ts

```diff
@@ -31,6 +31,8 @@ function ensureShapeOfTrackedProperties(trackedAnalytics?: Partial<ITrackedPrope
     sdkCode: 'T', // Base code
     sdkSemver : packageVersion.split('-')[0], // remove -beta, -alpha or other tagged versions from the version string
     product: 'A',
+    osType: 'Z',
```

Comment:
Don't we want to actually detect it?

Reply:
I think it'll be dependent on the inheriting  framework
For example, React Native will be the one to detect the platform and the os version.

Reply:
This package can be used as a standalone package in your pure JS project. We can detect os type and version.